.storybook/stories/forms/forms-required-mark.stories.ts

```diff
@@ -26,12 +26,13 @@ export default {
   ],
   argTypes: {
     // inputs
-    clrLabelSize: { control: { type: 'number', min: 1, max: 12 } },
+    clrLabelSize: { type: 'number', min: 1, max: 12 },
```

Comment:
```suggestion
    clrLabelSize: { control: { type: 'number', min: 1, max: 12 } },
```

This is wrong. Please revert.